util/src/kvdb.rs

```diff
@@ -598,4 +651,12 @@ mod tests {
 		let _ = Database::open_default(path.as_path().to_str().unwrap()).unwrap();
 		test_db(&DatabaseConfig::default());
 	}
+
+	#[test]
+	fn auto_compaction() {
```

Comment:
The test does not use any i/o, it just tests the `rotational_from_df_output` function. Not sure it should rely on system/software (but maybe can do using feature).